server/methods.coffee

```diff
@@ -67,14 +67,15 @@ Meteor.methods
           featureClass: geoname.featureClass
           featureCode: geoname.featureCode
           alternateNames: geoname.alternateNames
-      locationAnnotations = locationAnnotations.filter (loc)->
+      locationAnnotations.forEach (loc)->
         geoname = geonamesById[loc.geoname.geonameid]
         if geoname
           loc.geoname = geoname
```

Comment:
Wondering if this is this necessary? Maybe we just do what's in the else block if there's no `geoname`?